modules/OFConnectionManager2/module/src/cxn_instance.c

```diff
@@ -1987,6 +1997,7 @@ ind_cxn_alloc(controller_t *controller, uint8_t aux_id, int sock_id)
         cxn->keepalive.threshold = 0;
     }
     cxn->keepalive.tx_echo_cnt = 0;
+    cxn->async_pending_cnt = 0;
```

Comment:
i forgot to comment on this previously -- maybe the cxn's async_pending_cnt should be renamed to "pending_cnt", since there are no synchronous pending operations.